scripts/globals/status.lua

```diff
@@ -2333,24 +2333,25 @@ ANIMATION_MOUNT                    = 85;
 MOUNTS =
 {
     CHOCOBO        = 0,
-    RAPTOR         = 1,
-    TIGER          = 2,
-    CRAB           = 3,
-    RED_CRAB       = 4,
-    BOMB           = 5,
-    RAM            = 6,
-    MORBOL         = 7,
-    CRAWLER        = 8,
-    FENRIR         = 9,
-    BEETLE         = 10,
-    MOOGLE         = 11,
-    MAGIC_POT      = 12,
-    TULFAIRE       = 13,
-    WARMACHINE     = 14,
-    XZOMIT         = 15,
-    HIPPOGRYPH     = 16,
-    SPECTRAL_CHAIR = 17,
-    SPHEROID       = 18
+    QUEST_RAPTOR   = 1,
```

Comment:
that explains the off by one we were discussing

odd that the quest one needed its own id.

Reply:
The old enums actually match what the client is sending in 0x01A. When dealing with animations and etc they pop quest raptor in after chocobo...

It might be better to leave the enums the way they were and add quest raptor only to status.lua, I think thats where it actually belongs.
Then I could just define mount_key_item_offset somewhere and add it to mounmt enum.

The quest mount has different behavior, whenever I start scripting the quest I will need access to the updated status.lua

Reply:
Either way a 1 needs to be added to the param coming from 0x01A if mountID greater than 1, before setting the power of the mount status effect, and so far I haven't needed to use enums.